src/clj/harja/kyselyt/muutos_kyselyt.sql

```diff
@@ -43,15 +43,29 @@ SELECT m.id,
        m.voimassa_alkaen <= (SELECT TO_DATE(:hoitokauden_alkuvuosi || '-10-01', 'YYYY-MM-DD'))
  GROUP BY m.id, m.versio, m.urakka, m.voimassa_alkaen, m.tyyppi, m.nimi, m.syy, m.kulu_kohdistus, m.luonnos;
 
---name: paivita-muutos!
+-- name: rahavarausten-toteumat
+SELECT rv.id, SUM(kk.summa) as toteumat
+  FROM kulu k
+           JOIN kulu_kohdistus kk ON k.id = kk.kulu
+           JOIN toimenpideinstanssi tpi ON kk.toimenpideinstanssi = tpi.id
+           JOIN rahavaraus rv ON kk.rahavaraus_id = rv.id
+           JOIN rahavaraus_urakka rvu ON rv.id = rvu.rahavaraus_id
+           JOIN urakka u ON rvu.urakka_id = u.id AND tpi.urakka = u.id
+ WHERE u.id = :urakka AND
```

Comment:
Voisi AND:in pudottaa alemmalle riville "River" tyyppisen sql formatoinnin noudattamiseksi.